src/main/java/ChatSession.java

```diff
@@ -0,0 +1,67 @@
+import java.util.ArrayList;
+
+public class ChatSession {
+    // Store subclasses of tasks
+    ArrayList<Task> taskList;
+
+    public ChatSession() {
+        this.taskList = new ArrayList<Task>();
+    }
+
+    public void startSession() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    public void endSession() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    public void addTask(Task task) {
+        // Create a new Task, append to taskList
+//        Task newTask = new Task(description);
```

Comment:
Maybe the indentation of these comments could be in line with the code! It seems a little messy here but small issue